src/j1client.test.js

```diff
@@ -33,13 +51,32 @@ beforeEach(async () => {
   polly = new Polly("JupiterOneClient tests", {
     adapters: ["node-http"]
   });
-  polly.server.any().intercept((req, res) => {
-    attempts++;
-    const shouldFailThisTime = attempts <= attemptsToFail;
 
-    res.status(shouldFailThisTime ? 401 : 200);
-    res.json(shouldFailThisTime ? {} : mockResponse);
-  });
+  polly.server
+    .any("https://jqs-deferred.s3.amazonaws.com/s3-deferred-pass/1")
+    .intercept((req, res) => {
+      res.status(200);
+      res.json(mockQueryV1Response);
+    });
+
+  polly.server
+    .any("https://jqs-deferred.s3.amazonaws.com/deferred/1")
+    .intercept((req, res) => {
+      const s3ResponsePass = true;
+
+      res.status(s3ResponsePass ? 401 : 200);
+      res.json(s3ResponsePass ? mockS3ResponsePass : mockS3ResponseFail);
```

Comment:
If `s3ResponsePass` is always `true`, why do we have the ternaries here?

Was the setting of `s3ResponsePass` to `true` intentional or was there supposed to be some check there?

Reply:
@charlieduong94 was planning on testing failure, but ended up not. I removed the redundant code for now.